qcfractal/cli/qcfractal_manager.py

```diff
@@ -64,83 +186,148 @@ def main(args=None):
     # Grab CLI args if not present
     if args is None:
         args = parse_args()
-
     exit_callbacks = []
-    args["adapter_type"] = "executor"
 
-    # Handle Dask adapters
-    # if args["adapter_type"] == "dask":
-    #     dd = cli_utils.import_module("distributed")
+    # Try to read a config file first
+    config_file = args["config_file"]
+    if type(config_file) is str:  # Cast to unified code for same list
+        config_file = [config_file]
```

Comment:
This is a case where a config file should be passed in explicitly I think.

Reply:
It is, I just have this engineered as such because `config_file` (the Manager Settings) defaults to search in the 2 default locations through 
`add_argument("config_file", nargs="?", type=str,
                        default=[os.path.join(os.path.realpath("."), MANAGER_CONFIG_NAME),
                                 os.path.join(os.path.expanduser("~"), ".qca", MANAGER_CONFIG_NAME)])`
which returns a list, but if the user feeds in one arg, it returns a string and casting to list by `list(x)` with a list/tuple is fine, but it will turn a string into a list of 1-char long elements. By doing this, the code below is then identically handled

Reply:
For a manager, like a server we do not want to have default configs. They must be passed in explicitly.